helm/polymesh/values.yaml

```diff
@@ -23,7 +23,7 @@ polymesh:
     -   compiled
     - --prometheus-external
     - --telemetry-url
-    -   "'wss://telemetry.polymesh.live:443/submit 0'"
+    -   "'wss://telemetry.polymesh.network:443/submit 0'"
```

Comment:
Suggest to remove the `--telemetry-url` completely so it will use the runtime default which is `wss://telemetry.polymesh.network`

Reply:
Thanks @F-OBrien